data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
I suggest first focusing on the Errors PR and getting that merged before updating this one (ie, force-pushing). Then focus on this one and get it merged before updating the others. An all-in-one PR isn't a bad thing generally, but in this case the sheer number of changes is pretty overwhelming, so that's a good reason to keep them split up in my opinion.